### PR TITLE
test: complete tests

### PR DESCRIPTION
**Pull request by Xstoudi**
- [x] Migration.ts
- [x] Model.ts

### CODE REVIEW

**Comment by Xstoudi** [comment]
Type error on static User methods. I monkey-patched it by casting as any temporarily. `(User as any)` should be replaced by `User`.

**Comment by Xstoudi** [comment]
(it's not)

File: src/Model/__tests__/Model.test.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    new Error('document notavalidid not found in users'),
+  );
+});
+
+test('saved changes should be sent to database', async () => {
+  const user = await User.create({
+    username: 'root4',
+    password: 'root',
+  });
+  user.password = 'rootroot';
```
Comment (stropitek) [inline src/Model/__tests__/Model.test.ts:93]: You could check that the user's id is a ObjectId

File: src/Model/__tests__/Model.test.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+import { Mongodb } from '../../Mongodb';
+import { Model } from '../Model';
+
+interface IUser {
+  username: string;
+  password: string;
+}
+
+class User extends Model implements IUser {
```
Comment (stropitek) [inline src/Model/__tests__/Model.test.ts:11]: You can add tests for the IncrementalModel

**Review by stropitek — APPROVED** [review]

File: src/Model/__tests__/Model.test.ts (hunk 1 above)
Comment (Xstoudi) [reply src/Model/__tests__/Model.test.ts:110]: Done

File: src/Model/__tests__/Model.test.ts (hunk 2 above)
Comment (Xstoudi) [reply src/Model/__tests__/Model.test.ts:17]: Done

**Pull request merged**